app/src/main/java/com/example/reply/ui/ReplyHomeScreen.kt

```diff
@@ -95,7 +100,12 @@ fun ReplyHomeScreen(
         val navigationDrawerContentDescription = stringResource(R.string.navigation_drawer)
         PermanentNavigationDrawer(
             drawerContent = {
-                PermanentDrawerSheet(Modifier.width(dimensionResource(R.dimen.drawer_width))) {
+                PermanentDrawerSheet(
```

Comment:
`PermanentDrawerSheet` doesn't work well when its container color is customized by Modifier. Remove all `Modifier..background(MaterialTheme.colorScheme.inverseOnSurface)` and use the `drawerContainerColor` parameter like this:
```
                PermanentDrawerSheet(
                    modifier = Modifier
                        .width(dimensionResource(R.dimen.drawer_width)),
                    drawerContainerColor = MaterialTheme.colorScheme.inverseOnSurface
                ) {
                    NavigationDrawerContent(
                        selectedDestination = replyUiState.currentMailbox,
                        onTabPressed = onTabPressed,
                        navigationItemContentList = navigationItemContentList,
                        modifier = Modifier
                            .wrapContentWidth()
                            .fillMaxHeight()
                            .padding(dimensionResource(R.dimen.drawer_padding_content))
                    )
                }
```

Reply:
And you can remove `windowInsets =` as well.

Reply:
Thank you @yaraki